app/views/grants/_form.html.haml

```diff
@@ -1,130 +1,140 @@
+.grid-x
+  .cell.small-12.medium-3
+    = f.label :review_open_date, 'Review Open Date', class: 'middle'
+  .cell.small-12.medium-9
+    = f.text_field :review_open_date, value: date_mmddYYYY(grant.review_open_date), class: 'small-12 medium-6'
+.grid-x
+  .cell.small-12.medium-3
+    = f.label :review_close_date, 'Review Close Date', class: 'middle'
+  .cell.small-12.medium-9
+    = f.text_field :review_close_date, value: date_mmddYYYY(grant.review_close_date), class: 'small-12 medium-6'
+.grid-x
+  .cell.small-12.medium-3
+    = f.label :panel_date, 'Panel Date', class: 'middle'
+  .cell.small-12.medium-9
+    = f.text_field :panel_date, value: date_mmddYYYY(grant.panel_date), class: 'small-12 medium-6'
+.grid-x
+  .cell
+    = f.label :panel_location, 'Panel Location'
+    = f.text_area :panel_location, cols: 15, rows: 5, placeholder: 'Required if Panel Day is set. Use TBD if unknown.'
+.grid-x
+  .cell
     %h3
-      Panel and Reviews
-    .grid-x
-      .cell
-        = f.label :review_guidance, 'Review Guidance'
-        = f.hidden_field :review_guidance, id: 'grant_review_guidance'
-        %div
-          %trix-editor(input='grant_review_guidance')
-    .grid-x
-      .cell
-        = f.label :max_reviewers_per_proposal, 'Maximum Reviewers / Proposal', class: 'middle'
-        = f.number_field :max_reviewers_per_proposal, min: 1
-    .grid-x
-      .cell
-        = f.label :max_proposals_per_reviewer, 'Maximum Proposals / Reviewer', class: 'middle'
-        = f.number_field :max_proposals_per_reviewer, min: 1
-    .grid-x
-      .cell.small-12.medium-3
-        = f.label :review_open_date, 'Review Open Date', class: 'middle'
-      .cell.small-12.medium-9
-        = f.text_field :review_open_date, value: date_mmddYYYY(@grant.review_open_date), class: 'small-12 medium-6'
-    .grid-x
-      .cell.small-12.medium-3
-        = f.label :review_close_date, 'Review Close Date', class: 'middle'
-      .cell.small-12.medium-9
-        = f.text_field :review_close_date, value: date_mmddYYYY(@grant.review_close_date), class: 'small-12 medium-6'
-    .grid-x
-      .cell.small-12.medium-3
-        = f.label :panel_date, 'Panel Date', class: 'middle'
-      .cell.small-12.medium-9
-        = f.text_field :panel_date, value: date_mmddYYYY(@grant.panel_date), class: 'small-12 medium-6'
-    .grid-x
-      .cell
-        = f.label :panel_location, 'Panel Location'
-        = f.text_area :panel_location, cols: 15, rows: 5, placeholder: 'Required if Panel Day is set. Use TBD if unknown.'
-    .grid-x
-      = f.hidden_field :organization_id, value: current_user.organization.id
-      .actions
-        = f.submit 'Save and Complete', class: 'button'
-        = f.submit 'Save as Draft', class: 'button warning', name: 'draft'
-        - if @grant.persisted?
-          = link_to 'Cancel', @grant, class: 'button secondary'
-        - else
-          = link_to 'Cancel', grants_path, class: 'button secondary'
+      Applicant Questions
```

Comment:
I am not sure if this is the best heading here. But I am also not sure if it matters now.

Reply:
Agreed. I figure this will be a partial at some point.